ingredient/ingredient-function-app/README.md

```diff
@@ -0,0 +1,76 @@
+## Follow these basic steps to setup a new ingredient project:
```

Comment:
Remove the base package content, and you should at least include instructions for the parameters in your ingredient. Check some of the other ingredient readmes for format.

Reply:
I just got around to updating the readme